### ISSUE TITLE
in some cases (subscription mainly) the error response payload is plain text instead of a json object

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#2106]
In some cases (subscription mainly) the error response payload is plain text instead of a json object. The http code is 200 and it should be 400. 

See issues: #1989 #1991 #1994 #1996

#### received response
```
http code: 200
date: Fri, 29 Apr 2016 12:55:14 GMT
fiware-correlator: 9e01cd5c-0e09-11e6-a6c4-005056a20feb
connection: Keep-Alive
content-type: application/json
content-length: 40
payload: invalid character found in URI param /q/
```

#### expected response
```
http code: 400
date: Fri, 29 Apr 2016 12:55:14 GMT
fiware-correlator: 9e01cd5c-0e09-11e6-a6c4-005056a20feb
connection: Keep-Alive
content-type: application/json
content-length: 79
payload: {"error":"BadRequest","description":"invalid character found in URI param /q/"}

```

**Comment by fgalan** [comment]
Fixed by PR https://github.com/telefonicaid/fiware-orion/pull/2130

**Comment by fgalan** [comment]
Please @iariasleon have a look in order to check if this has been fixed (not the "parent" #1989 #1991 #1994 #1996, but the present one about returning JSON intead of text plain; I understand that other issues, e.g. correct status code or correct JSON fields in the response are the ones dealt at #1989 #1991 #1994 #1996 and will be fixed/checked in that scope).

**Comment by iariasleon** [comment]
LGTM, see issues: #1989 #1991 #1994 #1996
